roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -20,11 +20,18 @@
     - devices
   changed_when: false
   failed_when: false
+  register: activate_osd_disk
   when:
     not item.0.get("skipped") and
     item.0.get("rc", 0) != 0 and
     not osd_auto_discovery
 
+- name: Fail if ceph-disk cannot create an OSD
```

Comment:
just to stay consistent can you do: `s/Fail/fail/`?
Thanks!

The rest looks good :)